app/components/review-point.jsx

```diff
@@ -0,0 +1,237 @@
+  },
+
+  container: {
+    fontFamily: theme.font.fontFamily,
+    overflowX: 'hidden',
+  },
+  statusBadge: {
+    backgroundColor: theme.colors.statusBadgeProgressBackground,
+    border: `${theme.border.width.thin} solid ${theme.colors.statusBadgeProgressBorder}`,
+    ...sharedStatusBadgeStyle(),
+    alignSelf: 'flex-start',
+  },
+  statusBadgeComplete: {
+    backgroundColor: theme.colors.statusBadgeCompletedBackground,
+    border: `${theme.border.width.thin} solid ${theme.colors.statusBadgeCompletedBorder}`,
+    ...sharedStatusBadgeStyle(),
+    alignSelf: 'flex-start',
+  },
+  SvgContainer: {
+    position: 'absolute',
+    top: '0.3rem',
+    right: '0rem',
+    fontSize: '1.5rem',
+  },
+  subjectStyle: {
+    ...theme.font,
+    fontSize: '1.5rem',
+    fontWeight: '400',
+    lineHeight: '1.875rem',
+  },
+  descriptionStyle: {
+    ...theme.font,
+    alignSelf: 'stretch',
+    fontSize: '1rem',
+    fontWeight: '400',
+    lineHeight: '1.5rem',
+  },
+  contentContainer: {
+    padding: '2rem 1.875rem',
+    paddingRight: '15%',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      padding: '2rem 1.875rem',
+    },
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    gap: '1rem',
+    position: 'relative',
+    width: '100%',
+    boxSizing: 'border-box',
+    alignSelf: 'stretch',
+  },
+  showDualPointListContainer: {
+    padding: '0rem 1.875rem 1.875rem',
+    paddingRight: '15%',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      padding: '0rem 0rem 1.875rem',
+    },
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    gap: '1rem',
+    position: 'relative',
+    width: '100%',
+    boxSizing: 'border-box',
+    alignSelf: 'stretch',
+  },
+  informationGrid: {
+    display: 'grid',
+    gridTemplateColumns: '1fr', // Initially, set to 1 column
+    gap: '1rem',
+    width: '100%',
+  },
+
+  informationColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '1rem',
+    position: 'relative',
+    width: '100%', // Initially set to 100%
+  },
+
+  rankingColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '0.625rem',
+    position: 'relative',
+    width: '100%', // Initially set to 100%
+  },
+
+  // Additional style for adjusting grid layout
+  '@media (min-width: 40rem)': {
+    // Adjust breakpoint as needed
+    informationGrid: {
+      gridTemplateColumns: '1fr 0.25fr', // Adjusted grid layout
+    },
+  },
+
+  ranking: {
+    width: '70%',
+    fontSize: '1rem',
+    '& label': {
+      fontWeight: 'normal',
+    },
+  },
+  showDualPointList: {
+    width: '100%',
+    padding: '0rem',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      border: 'none',
+      '& > div': {
+        borderTopLeftRadius: '0rem',
+        borderTopRightRadius: '0rem',
+      },
+    },
+  },
```

Comment:
add:
```
'& hover':{
outline: 'none'
}
```
and that will get rid of the green border.

Reply:
To remove the green border on points when hovered, I updated the show-dual-point-list component, which is rendered within the review-point component, instead of modifying the review-point component itself.